client/query.go

```diff
@@ -2,7 +2,11 @@ package client
 
 import (
 	"context"
+	"encoding/hex"
+	"errors"
 	"fmt"
+	ctypes "github.com/tendermint/tendermint/rpc/core/types"
```

Comment:
Not sure what IDE you're using, but you should be able to configure it to use https://pkg.go.dev/golang.org/x/tools/cmd/goimports instead of gofmt to group imports properly.

Reply:
Thanks for that!

Reply:
I need to set this up too